src/main/scala/net/snowflake/spark/snowflake/Parameters.scala

```diff
@@ -173,6 +173,13 @@ object Parameters {
   val PARAM_INTERNAL_SKIP_WRITE_WHEN_WRITING_EMPTY_DATAFRAME: String = knownParam(
     "internal_skip_write_when_writing_empty_dataframe"
   )
+  // Internal option to use COPY command to write a DataFrame
+  // without any partitions. By default, it is 'true'.
+  // This option may be removed without any notice in any time.
+  // Introduced since Spark Connector 2.9.3
+  val PARAM_INTERNAL_USE_COPY_TO_WRITE_EMPTY_DATAFRAME: String = knownParam(
```

Comment:
I don't think we need this parameter to control it. do we?

Reply:
Yes, this is a bug fixing, and the possibility to this the problem is low. I will remove this parameter.